requirements.txt

```diff
@@ -1,3 +1 @@
-requests
-simplejson
-nose
+requests==2.5.0
```

Comment:
Let's make this >= 2.0.0 so that we give our clients some choice what lib to use.